docs/workshop/LAB-MANUAL.md

```diff
@@ -1,16 +1,17 @@
 ## Azure Credentials:
 
-**Username/email:** ++@lab.CloudPortalCredential(User1).Username++
-
-**Password:** ++@lab.CloudPortalCredential(User1).Password++
-
-**AzureEnvName:** ++AITOUR@lab.LabInstance.Id++
-
-**Subscription:** ++@lab.CloudSubscription.Id++
-
+```
```

Comment:
I'm not sure if this will render correctly in the lab manual. What is this change doing? @cedricvidal

Reply:
Actually, I just looked at the notebook and can understand what you were doing. Let me update this so it renders correctly and people could just copy the block and past in the cell!

Reply:
Ok, feel free to commit the fix